modules/ROOT/pages/setting-inaccessible-fields.adoc

```diff
@@ -0,0 +1,12 @@
+= Setting Inaccessible Fields 
```

Comment:
I think that we could be more precise with a title like "Inaccesible field handling" or something that states that these properties define the Debugger's behaviour regarding inaccesible fields (not totally sure about this one, so you can let as it is if you think is better).0.

Reply:
I like your suggestion! Thanks!